multus/multus.go

```diff
@@ -71,21 +71,17 @@ func loadNetConf(bytes []byte) (*types.NetConf, error) {
 		netconf.ConfDir = defaultConfDir
 	}
 
-	if netconf.UseDefault {
-		if netconf.Kubeconfig == "" || !defaultcninetwork {
-			return nil, fmt.Errorf(`If you have set always_use_default, you must also set the delegates & the kubeconfig, refer to the README`)
-		}
-		return netconf, nil
-	}
-
-	if !netconf.UseDefault && (netconf.Kubeconfig != "" && !defaultcninetwork) {
-		return netconf, nil
+	if netconf.Kubeconfig == "" || !defaultcninetwork {
+		return nil, fmt.Errorf(`You must also set the delegates & the kubeconfig, refer to the README`)
 	}
 
 	if len(netconf.Delegates) == 0 && !defaultcninetwork {
 		return nil, fmt.Errorf(`delegates or kubeconfig option is must, refer README.md`)
 	}
 
+	// default network in multus conf as master plugin
+	netconf.Delegates[0]["masterplugin"] = true
```

Comment:
So this PR makes the first delegate in the Multus CNI JSON config the masterplugin that returns its IP to Kubernetes, right?

Reply:
Correct. First network in /etc/cni/net.d/*.conf will be masterplugin (e.g. default network).